syntax = "proto3";

package msg;

message TaskMsg {
  repeated string toExecute = 1;
}
